drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -149,8 +149,9 @@ Error AudioDriverPulseAudio::init_device() {
 			break;
 
 		default:
-			ERR_PRINTS("PulseAudio: Unsupported number of channels: " + itos(pa_channels));
-			ERR_FAIL_V(ERR_CANT_OPEN);
+			pa_channels = 2;
+			channels = 2;
+			WARN_PRINTS("PulseAudio: Unsupported number of channels: " + itos(pa_channels));
```

Comment:
You should issue the warning before overriding `pa_channels`, otherwise it'll say that 2 is an unsupported number of channels :)

Reply:
Nice catch, just fixed it :).